.github/workflows/presubmit.yml

```
@@ -3,11 +3,12 @@ on: [push, pull_request]
 
 jobs:
   build:
-    name: Build ${{ matrix.os }} ${{ matrix.arch }}
+    name: Build ${{ matrix.os }} ${{ matrix.arch }}${{ matrix.extra }}
```

What's the reason for not doing
```suggestion
    name: Build ${{ matrix.os }} ${{ matrix.arch }} ${{ matrix.extra }}
```
and avoiding the leading space in the `extra:` specifications below?  I fear the leading space might be easily overlooked/forgotten.

---

To avoid including a trailing space in the name where there matrix.extra is unset.